src/api/rpc_api/author.rs

```diff
@@ -14,15 +14,12 @@
 //! Interface to common frame system pallet information.
 
```

Comment:
Only for extrinsic submission now?

Reply:
Oh, good catch. That shouldn't be frame system 👍

Reply:
Updated :)

Reply:
Yes thanks